src/main/kotlin/lotto/LottoNumbers.kt

```diff
@@ -0,0 +1,27 @@
+package lotto
+
+class LottoNumbers(var lottoNumbers: List<LottoNumber>) {
```

Comment:
```suggestion
class LottoNumbers(val lottoNumbers: List<LottoNumber>) {
```
List, Map, Set과 같은 컬렉션은 add, remove등 내부적으로 데이터를 제어할 수 있어요.
컬렉션은 val을 활용해보면 어떨까요?

Reply:
@BeokBeok 이 부분은 `sorted` 하기 위해 어쩔 수 없는 부분 아닐까요?

Reply:
sort를 내부에서 하고, 그 값을 외부에서 접근이 가능하도록 변경해보면 어떨까요?
외부에서 새로운 List<LottoNumber>를 만들어서 lottoNumbers에 set을 하지 못하도록 변경해보면 어떨까요?

Reply:
`Set`이라는 자료형을 활용하셔도 좋습니다. 😄